frontend/src/scenes/notebooks/Nodes/NotebookNodeReplayTimestamp.tsx

```diff
@@ -75,6 +75,25 @@ export const NotebookNodeReplayTimestamp = Node.create({
     group: 'inline',
     atom: true,
 
+    serializedText:
+        () =>
+        (attrs: { playbackTime: number }): string => {
```

Comment:
Do the types not get carried over here?

Reply:
I missed that there was already an interface, so can use that... but (I think) there's no type detection here because we're not wrapped in the create widget function that carries the type